tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -36,7 +36,7 @@ def test_fields(self):
         self.assertEqual(uut.indent, ' ')
         self.assertEqual(str(uut), 'my doc')
         self.assertEqual(uut.marker, ('/**', '*', '*/'))
-        self.assertEqual(uut.range, (25, 45))
+        self.assertEqual(uut.position, (25, 45))
 
```

Comment:
This is test fails. I can't understand the objective of `def test_fields()`. Hence, I was unable to resolve it.
CC @Makman2 @NiklasMM